python/cog/director/director.py

```diff
@@ -258,8 +265,13 @@ def _handle_message(self, message_id, message_json) -> None:
         # predictions, but isn't exiting.
         if tracker.status == schema.Status.FAILED:
             self._record_failure()
+            span.set_attribute("prediction.status", 'success')
```

Comment:
You've got these statuses inverted I think. Given we can also have `canceled` as a status, I think it'd be cleaner to just use `span.set_attribute("prediction.status", tracker.status)` outside this conditional.